libs/feature/record/src/lib/ign-api-dl/ign-api-dl.component.ts

```diff
@@ -0,0 +1,249 @@
+import {
+  ChangeDetectionStrategy,
+  Component,
+  Input,
+  OnInit,
+} from '@angular/core'
+import { DatasetServiceDistribution } from '@geonetwork-ui/common/domain/model/record'
+import { BehaviorSubject, Observable, combineLatest, map, mergeMap } from 'rxjs'
+import { HttpClient } from '@angular/common/http'
+import { Choice } from '@geonetwork-ui/ui/inputs'
+import axios from 'axios'
+
+export interface Label {
+  label: string
+}
+
+export interface FormatProduit {
+  title: string
+  update: string
+  format: Array<TermBucket>
+  zone: Array<TermBucket>
+}
+
+export interface FormatSortieProduit {
+  label: string
+  value: string | number
+}
+export interface ListUrl {
+  url: string
+}
+
+export interface listChoice {
```

Comment:
I would also change this to upper case `ListChoice`. Just to be consistent :)

Reply:
Done